packages/boa/.eslintrc.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
     "capitalized-comments": "off",
     "class-methods-use-this": "error",
     "comma-dangle": "off",
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I think we don't need this because the typescript rule won't have effect on JavaScript files.

Reply:
When I run `npm run lint` for boa project, the same error will be detected by both `@typescript-eslint/no-unused-vars` and `no-unused-vars`.

Reply:
```shell
$ npm run lint
> @pipcook/boa@1.0.3 lint /Users/x/pipcook/packages/boa
> eslint . && ./clang-format.py

/Users/x/pipcook/packages/boa/tests/base/basic.js
  131:12  warning  '_' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

Reply:
How about removing the `_` as the warning tip?

Reply:
I mean the plugin `@typescript-eslint/no-unused-vars` was applied to the js code.
And I think this test case can't be fixed if we need to check the array deconstruction of the builtin object. 
```js
t.throws(() => {
    // eslint-disable-next-line no-unused-vars
    const [_] = builtins;
    // Should throw the error
  }, TypeError);
```

Reply:
We can just print the destructed constants via:

```sh
console.log(_);
```